fairlearn/datasets/__init__.py

```diff
@@ -7,8 +7,10 @@
 
 from ._fetch_adult import fetch_adult
 from ._fetch_boston import fetch_boston
+from ._fetch_bank_marketing import fetch_bank_marketing
 
 __all__ = [
     "fetch_adult",
     "fetch_boston",
+    "fetch_bank_marketing",
```

Comment:
Can you add this to the dataset tests?

Reply:
Yes, will do after https://github.com/fairlearn/fairlearn/pull/533 merges as this one has the tests.

Reply:
I merged #533

Reply:
Added the tests